src/docs/build_files.md

```diff
@@ -149,8 +143,8 @@ techniques can be especially helpful:
 *What targets does a BUILD file define?* Use the `list` goal:
 
     :::bash
-    $ ./pants list examples/src/java/org/pantsbuild/example/hello/greet
-    examples/src/java/org/pantsbuild/example/hello/greet:greet
+    $ ./pants list src/java/myproject/example/hello/greet
```

Comment:
I know it's convention to do `src/java/com/myorg/myproject`. Lmk if I should use that name instead. 

The verbosity kills me, though. Maybe we should change this to `src/python/myproject/example`.

Reply:
I think the `examples/src/java/org/pantsbuild/example/hello/geet` target actually exists (meaning running this command will actually work) Not sure that's really necessary though.

Reply:
> I know it's convention to do src/java/com/myorg/myproject. Lmk if I should use that name instead.

Yea, you should. This is a thing, for better or worse.

That, or switch to describing python.